WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/AboutViewController.swift

```diff
@@ -74,6 +79,13 @@ private extension AboutViewController {
         tableView.estimatedRowHeight = Constants.rowHeight
         tableView.rowHeight = UITableView.automaticDimension
         tableView.backgroundColor = .listBackground
+
+        guard let tabBarHeight = tabBarController?.tabBar.frame.height else {
+            return
+        }
```

Comment:
non-blocking question: just wondering, could the tab bar height change over the view controller life cycle?

Reply:
You are right. Yes, it can change, for example during the device rotation. I updated the code to manage it 🙇